app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [OPTIONAL] As you use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_create` to call the `#update_comments_counter` method every time a new comment has been created to the database, then make sure the `update_comments_counter` method is private 🙃 🙃